### DISCUSSION TITLE
libvips meetup, London, Wednesday 29 November 2023

### DISCUSSION THREAD

**Discussion by lovell** [libvips/libvips#3748] [General]
We (@jcupitt, @kmartinez and myself) are planning to meet on the 29th of November in central London and anyone that uses or is interested in using libvips will be very welcome to join us for free (OpenCollective-funded) food, drink and informal chat.

We're meeting at [The Lyric](https://www.lyricsoho.co.uk/) from 6:30pm as Soho seems to work for most people.

Please let us know below if you're able to attend and subscribe for updates as we make them.

**Comment by lovell** [comment]
We're organising food for this Wednesday so please add a comment here if you're able to attend as we need to book for the correct number of people. Thank you!

**Reply by jcupitt** [reply]
Brian Millar, an old friend of mine, would like to join us as well, so +1 more.

**Reply by lovell** [reply]
Great, all welcome.

How about we meet from 6:30pm at the Lyric in Soho (5 mins from Piccadilly Circus tube) then perhaps eat around 8pm. Darjeeling Express (Kingly Court) or Masala Zone (the new one, next to Criterion) both good food options nearby but we might struggle to get a table for us all at the former.

**Reply by kmartinez** [reply]
OK! we need to try booking Darj Express I think.. so we can decide

**Reply by jcupitt** [reply]
Shall I try and book Darjeeling Express for five people?

**Reply by jcupitt** [reply]
Wait, have you changed your mind Ingvar Stepanyan? In which case it's four.

**Reply by RReverser** [reply]
I'm trying to see if I can get out of a meeting 😅 It's ironic that this happens 2nd time in a row exactly on the day of libvips meetup.

**Reply by jcupitt** [reply]
@lovell we should book! Shall I do it, or are you doing it?

**Reply by lovell** [reply]
Masala Zone have a table available at 8:30pm for 6. (We'll have to split over 2 tables at Darjeeling, plus there's £50pp deposit.)

**Reply by jcupitt** [reply]
Let's go for masala zone then. It's not very glamorous, but it's not too bad either.

**Reply by lovell** [reply]
Table for 6 at 8:30pm booked at the new-ish [Piccadilly Circus Masala Zone](https://www.masalazone.com/locations/piccadilly-circus/), which is the old [Criterion Restaurant](https://en.wikipedia.org/wiki/Criterion_Restaurant) so it'll be opulence over glamour ;)

**Reply by jcupitt** [reply]
Oooo I've never been to the Criterion! Now I'm excited again -- see you tonight, and thank you for booking.

**Reply by jcupitt** [reply]
... Brian has cancelled, so we're down to five again.

**Reply by RReverser** [reply]
FWIW I can make it after all. See you there!

**Reply by RReverser** [reply]
@lovell Could you please email me some alternative contact details (since you have my email anyway)? Not sure I'll be make it by 6:30pm, so would be good to have more details to find each other.

**Reply by jcupitt** [reply]
I had a great time! It was good to meet in person again, and the curry was very nice.

**Comment by lovell** [comment]
Thank you to everyone that has donated to the libvips OpenCollective - here we are spending a very small quantity of it as carefully as possible.

![](https://github.com/libvips/libvips/assets/210965/0ad3abd6-91d2-441a-b2d1-0d0355e4d624)

(Thanks Ingvar for the photo.)

**Comment by kmartinez** [comment]
As always it was great to meet up and catch-up with everyone's stories!
